spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -418,32 +418,32 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
 
   describe('choosing and unchoosing', () => {
```

Comment:
It may be even simpler and more future-proof if we go one step further and have:

```
describe('choosing and unchoosing' () => { ... });

describe('choosing and unchoosing for multiple', () => { ... });
```

In real use scenarios we don't expect people to be turning multiple on and off dynamically, so we should not be doing so in our tests either.